Makefile

```diff
@@ -446,6 +446,11 @@ override CFLAGS   := $(MK_CPPFLAGS) $(CPPFLAGS) $(MK_CFLAGS) $(CFLAGS)
 override CXXFLAGS := $(MK_CPPFLAGS) $(CPPFLAGS) $(MK_CXXFLAGS) $(CXXFLAGS)
 override LDFLAGS  := $(MK_LDFLAGS) $(LDFLAGS)
 
+COMMA := ,
```

Comment:
Why are you defining a variable for this?

Reply:
This is a common Makefile pattern. It is needed because a literal comma would be interpreted as an argument separator in the below `subst` line.
nvcc also accepts e.g. `-Xcompiler "-flag1 -flag2"` which may look cleaner.